software/software.mdown

```diff
@@ -25,13 +26,3 @@ JMAP is still young but several implementations already exist, and more are on t
 * [Dovecot](http://dovecot.org/pipermail/dovecot/2016-November/106262.html)
 * [Roundcube Next](https://github.com/search?q=org%3Aroundcube-next+jmap&type=Code)
 
-
-## Requested
-
-* [RainLoop Webmail](https://github.com/RainLoop/rainloop-webmail/issues/1378)
-* [Cypht](https://github.com/jasonmunro/cypht/issues/180)
-* [Horde IMP Webmail](https://bugs.horde.org/ticket/14683)
-
-
-
-
```

Comment:
Did you mean to remove the "Requested" section?

Reply:
I broke the file, sorry.